earth_enterprise/src/fusion/autoingest/AssetFactory_unittest.cpp

```diff
@@ -27,49 +28,13 @@
 using namespace std;
 using namespace AssetFactory;
 
-// forward declare classes
```

Comment:
I essentially rewrote the MockAsset class hierarchy from scratch. I tried to keep parts from the original version but that probably won't show up in the diff. The tests themselves should be more similar, but I did make some changes, including getting rid of duplicate tests and adding new tests to get better coverage.